src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml

```diff
@@ -2445,10 +2447,19 @@
             </Trigger>
         </ControlTemplate.Triggers>
     </ControlTemplate>
-
+    
+    <DataTemplate x:Key="expanderHeader">
+        <TextBlock
+            Text="{Binding}" 
+            FontSize="14"
+            FontWeight="Medium">
```

Comment:
I think this will probably do something once we change all the fonts to Artifakt.